cmd/gateway/main.go

```diff
@@ -40,6 +40,8 @@ type GatewayFlags struct {
 	uplink.Config
 
 	Version checker.Config
+
+	PBKDFConcurrency int `help:"please see <url>. default value recommended" default:"0"`
```

Comment:
Do we need to put an actual URL value here?

Reply:
yes, we also need to update the comment about which version numbers the problem applies to, once we have a version it no longer applies to. i'll submit a new pr with updates for both when that happens